meta-lmp-base/recipes-bsp/alsa-state/alsa-state/alsa-state-init

```diff
@@ -0,0 +1,43 @@
+#! /bin/sh
+#
```

Comment:
Do we use this script at all? Since we're not using sysvinit. I know it could be called by systemd, but curious if that is the current use case.

Reply:
It runs only in sysvinit systems. Well, if we want to try to upstream such solution, it should generic, not only for systemd systems.